### ISSUE TITLE
Use `FastJet.jl` for comparison later

### ISSUE THREAD

**Issue by Moelf** [gojakuch/JetReconstruction.jl#1]
it looks like eventually we'd compare to FastJet's output, @jstrube has kindly wrapped the C++ library in Julia: https://github.com/jstrube/FastJet.jl . I also assume Jan is happy to help you with any missing features there.

**Comment by jstrube** [comment]
Happy to help indeed. With more collaborators it may make sense to move FastJet.jl to JuliaHEP for maintenance

**Comment by gojakuch** [comment]
Yes, comparison with FastJet is what has been planned to ensure correctness of the code. It is good to know that there's a wrapper for Julia. Once there's a working naive implementation, the focus will be shifted to developing a more intelligent and optimised approach.
